cpp/src/arrow/compute/kernels/scalar_temporal_test.cc

```diff
@@ -118,6 +118,30 @@ class ScalarTemporalTest : public ::testing::Test {
           "2010-01-04T07:35:36", "2006-01-01T08:40:39",
           "2005-12-31T09:45:45", "2008-12-27T23:59:59",
           "2008-02-28", "2012-03-01", null])";
+  const char* durations_s =
```

Comment:
Nit: you could also reuse a set already in this file.

Reply:
Thanks for reminding. Actually I was planning to do so at the beginning. However I found the existing times and dates sets lack either duration ranges over 1 day or the subtle cases of negative duration, of which special care must be taken.

Let me know if my understanding is wrong.

Reply:
That's a good enough reason to not use it then :D